machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts

```diff
@@ -1260,6 +1258,20 @@ export const ExistingMosipVCItemMachine =
           {to: context => context.serviceRefs.store},
         ),
 
+        closeViewVcModal: send('DISMISS_MODAL', {
+          to: () => getHomeMachineService(),
+        }),
+
+        removeTamperedVcItem: send(
```

Comment:
Remove this method as it is not getting used

Reply:
done